bigframes/session/__init__.py

```diff
@@ -1456,15 +1456,37 @@ def read_gbq_function(
             The return type of the function must be explicitly specified in the
             function's original definition even if not otherwise required.
 
+        BigQuery UDFs has many public functions under the ``bqutil`` project on publicly shared datasets
```

Comment:
I'm finding this sentence difficult to understand. Perhaps, something like this?

```suggestion
        BigQuery Utils provides many public functions under the ``bqutil`` GCP project
```